### ISSUE TITLE
Markdown tables styling isn't set

### ISSUE THREAD

**Issue by tuurep** [jannis-baum/vivify#56]
Hi again,

Today I was writing a Markdown table for the first time and noticed the styling's very crude - it's the same situation as with the blockquotes earlier

For example this markdown table:

```md
| Option | Description | Possible values |
| ------ | ----------- | --------------- |
| `foo`  | Bar         | baz             |

```

Renders like this with vivify:

![image](https://github.com/jannis-baum/vivify/assets/15036238/4ec694db-0128-415c-9707-f6f70a0ea238)

Github would style it like this:

![image](https://github.com/jannis-baum/vivify/assets/15036238/cd19f63e-b33a-4c76-96f0-8b137360626f)

I suggest we should once again copy the CSS github uses for tables. I can find out & send a PR if you agree.

Just have to remember that dark mode might need another color for the table border, but I'll see if another element uses the same color first.

**Comment by tuurep** [comment]
Oh yeah, my example screenshots don't show one important thing to note about Github md tables:

![image](https://github.com/jannis-baum/vivify/assets/15036238/21861ffa-c47b-4723-b00d-f33e8522032d)

The alternating background color every other row. Would be nice to have & probably doable as well.

**Comment by jannis-baum** [comment]
Hello!☺️

> I suggest we should once again copy the CSS github uses for tables. I can find out & send a PR if you agree.

This sounds great! Thank you!

**Comment by jannis-baum** [comment]
Closed in #58
